ledger/participant-integration-api/src/main/scala/platform/store/backend/common/CommonStorageBackend.scala

```diff
@@ -541,13 +545,11 @@ private[backend] trait CommonStorageBackend[DB_BATCH] extends StorageBackend[DB_
       queryOffset: Long,
   )(connection: Connection): Vector[(Offset, PackageLedgerEntry)] = {
     import com.daml.platform.store.Conversions.OffsetToStatement
-    SQL_GET_PACKAGE_ENTRIES
-      .on(
-        "startExclusive" -> startExclusive,
-        "endInclusive" -> endInclusive,
-        "pageSize" -> pageSize,
-        "queryOffset" -> queryOffset,
-      )
+
```

Comment:
@dasormeter would you be okay with not doing pure refactorings together with the meaty part? we can do this later, no functional change here, adds noise. I think a TODO comment to refactor these things is sufficient for now, and we can do refactorings later

Reply:
yeah this was a change when initially we thought the `SQL(...)` vs `SQL"..."` interpolation was causing an issue but it turned out that it was not needed -- i'll try to roll it back

Reply:
resolved